how_to_build.md

```diff
@@ -50,7 +49,7 @@ After the build has completed, the build artifacts will be located in `artifacts
 
 * View the full list of build targets:
 
-    `build.cmd -T`
+    `build.cmd -l`

```

Comment:
I usually use `-t`, which shows the dependencies between the targets.
![image](https://github.com/FakeItEasy/FakeItEasy/assets/1864281/82edb129-09c3-4e1c-8308-61131cacc97a)

I guess showing the flat list is fine, if that's what you need, but we might want to mention `-t` as well

Reply:
I don't mind switching to `-t` if you prefer. I'd rather not list both though. We're not here to duplicate bullseye's docs…